### DISCUSSION TITLE
Permission denied

### DISCUSSION THREAD

**Discussion by MaheshReddyYarrappagari** [termux/termux-packages#17511] [General]
./linux.sh
bash: ./linux.sh: Permission denied
How to solve this

**Comment by truboxl** [comment]
chmod +x ./linux.sh

**Reply by MaheshReddyYarrappagari** [reply]
Next step

**Reply by truboxl** [reply]
You can already ./linux.sh

**Comment by MaheshReddyYarrappagari** [comment]
It is not accepting
./linux.sh
![Screenshot_2023-07-25-21-02-46-77_84d3000e3f4017145260f7618db1d683](https://github.com/termux/termux-packages/assets/140540807/7c4d6120-8eab-48d9-a9fb-2c3908b6d743)

**Reply by agnostic-apollo** [reply]
https://wiki.termux.com/wiki/Hacking
